doc/sphinx/cligroup/__init__.py

```diff
@@ -2,5 +2,3 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-import pkg_resources
-pkg_resources.declare_namespace(__name__)
```

Comment:
The `doc\sphinx\` folder appears related to documentation generation but hasn't been updated in a long time.

Confirmed with doc team, they only use `doc\sphinx\azhelpgen\doc_source_map.json`